### ISSUE TITLE
Could you tell me how to run the model on real datasets?

### ISSUE THREAD

**Issue by GudonKing** [sunilkmaurya/GNN-Bet#1]
I tried to run the model on real datasets, but faild to reproduce the kt coefficient scores on real datsets from papers.

**Comment by sunilkmaurya** [comment]
Hi,
Thanks for your interest in our work. It is difficult to say where the problem could be.
This code is based on older version on PyTorch and there were some changes in Dropout
usage in new version that can cause some issues or there could be some issues with training
the model.
You can check out this repo: https://github.com/sunilkmaurya/GNN_Ranking and corresponding 
full paper for more details. The processing of real dataset is similar to the synthetic datasets.
